cmd/swarmctl/service/logs.go

```diff
@@ -23,6 +23,8 @@ var (
 			}
 
 			follow, err := cmd.Flags().GetBool("follow")
+			noStdout, err := cmd.Flags().GetBool("no-stdout")
+			noStderr, err := cmd.Flags().GetBool("no-stderr")
```

Comment:
Need to check `err` for both of these.

Maybe `stdout` and `stderr` that default to true are better flag names?

Are there any similar flags in Docker already?